src/typings.ts

```diff
@@ -686,6 +686,37 @@ export interface TrustlineAccruedInterestsObject {
   counterparty: string
 }
 
+export type UserAccruedInterestsRaw = TrustlineAccruedInterestsRaw[]
+export type UserAccruedInterestsObject = TrustlineAccruedInterestsObject[]
+
+export interface MediationFeeRaw {
+  value: string
+  from: string
+  to: string
+  transactionHash: string
+  timestamp: number
+}
+
+export interface MediationFeeObject {
+  value: Amount
+  from: string
+  to: string
+  transactionHash: string
+  timestamp: number
+}
+
+export interface EarnedMediationFeesListRaw {
+  user: string
+  network: string
+  mediationFees: MediationFeeRaw[]
+}
+
+export interface EarnedMediationFeesListObject {
+  user: string
+  network: string
+  mediationFees: MediationFeeObject[]
+}
+
```

Comment:
So, why do we need both Raw and Object definitions if they are 100% the same?

Reply:
Ah, `value: Amount` is the difference. hm...

Reply:
One of them is what you get from the relay server, the other one is what you provide to the user, with interpretation of the decimals to use to show the amount.